packages/component-library/draft/Kaizen/Modal/presets/ConfirmationModal.tsx

```diff
@@ -53,17 +66,15 @@ const ConfirmationModal = ({
             [styles.positiveHeader]: type === "positive",
             [styles.informativeHeader]: type === "informative",
             [styles.negativeHeader]: type === "negative",
+            [styles.cautionaryHeader]: type === "cautionary",
           })}
         >
           <div className={styles.iconContainer}>
             <svg className={styles.iconBackground}>
               <circle cx="75" cy="75" r="75" />
             </svg>
             <div className={styles.icon}>
-              <Icon
-                icon={type === "positive" ? success : information}
-                role="presentation"
-              />
+              <Icon icon={getIcon(type)} />
```

Comment:
Should this icon still have a role of presentation?

Reply:
Yes, all icons need to have a `role`. I think the build will even break if these are removed.

There's an explanation of what a role is and why it's important in the source code for the Icon component: `packages/component-library/components/Icon/Icon.tsx`.

Reply:
Good to know! FYI I noticed the builds did pass without it (I had accidentally deleted it before but added it back in a later commit)